src/frontend/lib/mapeo-schema.d.ts

```diff
@@ -0,0 +1,189 @@
+// Refer to https://github.com/digidem/mapeo-schema/blob/master/docs/README.md
+declare module "mapeo-schema" {
+  // TODO: do we need this?
```

Comment:
The idea is that all database types are an extension of this, e.g.

```ts
type Observation = Common & {...}
```

However presets and fields are not yet stored in the database, and we do not use `Node` and `Way` types here, so it's not that useful when the only type based on `Common` is `Observation`.

Reply:
Makes sense. can remove if it just adds noise for now (same for node and way types)

Reply:
Decided to keep it for now but open to removing if you have strong feelings on it